app/queries/admin/gdpr_delete_requests_query.rb

```diff
@@ -0,0 +1,19 @@
+module Admin
+  class GDPRDeleteRequestsQuery
```

Comment:
Since you're not instantiating an object (e.g. there's no `Admin::GDPRDeleteRequestsQuery.new`) please covert this to a module.  My understanding, from the sage ancestors of Forem's codebase is that a module is a bit less expensive for the interpreter to evaluate than a class.